checker/tests/test_Rules.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import AnDO_engine as ANDO
```

Comment:
shall we use everywhere AnDO (with these lower case and upper case letters)?
or maybe it's going to be painful to type?
but what's sure is that I don't like the "all upper case" version ANDO...
shall we replace it everywhere by ando (all lower cases)?
we keep AnDO_engine, but we import it as ando...
ok for you? if yes, can you replace ANDO by ando everywhere?

Reply:
i'll  replace "ANDO" by "ando"all lower case when importing